.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [12.x, 14.x]
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Versions
+        run: yarn versions
+
+      - name: Install dependencies
+        run: yarn install #--frozen-lockfile
+
+      # - name: Check
+      #   run: yarn check
+
+      # - name: Lint
+      #   run: yarn lint
+
+      - name: Audit dependencies
+        run: yarn audit --groups dependencies
+
+      # - name: Audit devDependencies
+      #   run: yarn audit --groups devDependencies
+
+      # - name: Audit peerDependencies
+      #   run: yarn audit --groups peerDependencies
+
+      # - name: Audit optionalDependencies
+      #   run: yarn audit --groups optionalDependencies
+
+      - name: Build
+        run: yarn build
+
+      - name: Test
+        run: yarn test
+
+      - name: Codecov
+        run: yarn codecov
```

Comment:
Codecov is requesting a repo token

Reply:
Just checked travis and I don't seem to have had the codecov token setup, maybe I just ran codecov to get a local coverage folder that I could use with code climate?

Reply:
Yes, maybe

We could also think about a general new code coverage solution at all

Reply:
Let's fix the codecov/codeclimate thing in a separate PR